tests/pytests/utils.py

```diff
@@ -77,8 +79,27 @@ def run_psi4_cli(inputs, outputs, extra_commands=None, as_binary=None):
 
     if extra_commands is None:
         extra_commands = []
-    psi_runner = os.path.join(os.path.dirname(os.path.abspath(psi4.__file__)), "run_psi4.py")
-    cmds = [sys.executable, psi_runner, "--inplace"] + extra_commands + list(inputs.keys())
+
+    cmds = None
+    psidir = os.path.dirname(os.path.abspath(psi4.__file__))
+
+    # Check inplace
+    psi_runner = os.path.join(psidir, "run_psi4.py")
+    if os.path.isfile(psi_runner):
+        cmds = [sys.executable, psi_runner, "--inplace"]
+
+    # Check test install
+    if cmds is None:
+        binpath = os.path.join(os.path.dirname(os.path.dirname(psidir)), "bin")
```

Comment:
regrettably, it won't work because in a proper python installation, there's the `pythonx.x/site-packages` in the way, which is of variable length. hence [the craziness](https://github.com/psi4/psi4/blob/master/psi4/run_psi4.py#L106). i'll take a stab at this.

Reply:
Ok, well the tests will simply skip for now. No harm.